src/Sylius/Bundle/ApiBundle/Provider/CustomerProvider.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Provider;
+
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Repository\CustomerRepositoryInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\User\Canonicalizer\CanonicalizerInterface;
+
+class CustomerProvider implements CustomerProviderInterface
+{
+    /** @var CanonicalizerInterface */
+    protected $canonicalizer;
```

Comment:
```suggestion
    private $canonicalizer;
```
same for the rest of properties